testing/README.md

```diff
@@ -12,21 +12,52 @@ Some UIs to check out:
 Testgrid (Test Results Dashboard): https://k8s-testgrid.appspot.com/sig-big-data
 Prow (Test-runner dashboard): https://prow.k8s.io/?repo=tensorflow%2Fminigo
 
-## Local testing
+## Updating continuous integration tests
 
-To test out changes to the docker image, first build the test-harness image:
+Build and push the test-harness image:
```

Comment:
when is this necessary?  What part of our TF build etc. would need to change to merit rebuilding the image?

Reply:
I just had to do it to install `clang-format` on the testing image.